wad2project/rotten_potatoes/views.py

```diff
@@ -205,10 +205,11 @@ def rate_movie(request, movie_name_slug):
     if request.method == "POST":
         form = AddRatingForm(request.POST)
         if form.is_valid():
-            rating = form.save(commit=False)
-            rating.user = request.user
-            rating.movie = Movie.object.get(slug=movie_name_slug)
-            rating.save()
+            rating_obj = form.save(commit=False)
+            rating_obj.user = request.user
+            rating_obj.movie = Movie.object.get(slug=movie_name_slug)
+            rating_obj.rating = int(form.rating)
+            rating_obj.save()
```

Comment:
assuming the rating will be done via some radio button, output should be number from 1 to 5 as a string. Thus, convert to int.